jetty-openid/src/main/config/etc/jetty-openid.xml

```diff
@@ -3,6 +3,8 @@
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
   <New id="OpenIdConfiguration" class="org.eclipse.jetty.security.openid.OpenIdConfiguration">
     <Arg><Property name="jetty.openid.openIdProvider"/></Arg>
+    <Arg><Property name="jetty.openid.openIdProvider.authorizationEndpoint"/></Arg>
+    <Arg><Property name="jetty.openid.openIdProvider.tokenEndpoint"/></Arg>
```

Comment:
I think the openIdProvider could be dropped from the property name.
`jetty.openid.authorizationEndpoint` and
`jetty.openid.tokenEndpoint`

Reply:
I was trying to relate them to the OP since there could be a growing set of properties for this module, some related to the OP in particular and some related to the module itself. If you don't feel strongly about this suggestion, I'll leave this.

Reply:
Okay this could be fine to keep, but I am thinking we should at least drop the `jetty.openid.openIdProvider` to `jetty.openid.provider` as the double `openid` is redundant.

```xml
<Property name="jetty.openid.provider" deprecated="jetty.openid.openIdProvider"/>
```

Also do you think `jetty.openid.issuer` would be a more accurate name for this property?

Reply:
Ideally, I think that `jetty.openid` is the best root. For the general settings, they should be placed there. For properties of the OP, I think that putting them under `jetty.openid.provider` would be best.

If we change these though, won't it be a breaking change? Should we allow the old and the new to try to avoid that?

Reply:
There is the deprecated property name to handle name changes without breakage.

Reply:
Made those changes. Thanks @gregw and @lachlan-roberts for pointing out this feature to me. Please just check that I used it right.